packages/hoppscotch-sh-admin/src/helpers/auth.ts

```diff
@@ -223,34 +163,19 @@ const refreshToken = async () => {
   }
 };
 
-async function elevateUser() {
-  const res = await axios.get(
-    `${import.meta.env.VITE_BACKEND_API_URL}/auth/verify/admin`,
-    {
-      withCredentials: true,
-    }
-  );
 
+const elevateUser = async () => {
+  const res = await authQuery.elevateUser();
   return !!res.data?.isAdmin;
 }
 
 async function sendMagicLink(email: string) {
-  const res = await axios.post(
-    `${import.meta.env.VITE_BACKEND_API_URL}/auth/signin?origin=admin`,
-    {
-      email,
-    },
-    {
-      withCredentials: true,
-    }
-  );
-
+  const res = await authQuery.sendMagicLink(email);
   if (res.data && res.data.deviceIdentifier) {
     setLocalConfig('deviceIdentifier', res.data.deviceIdentifier);
   } else {
     throw new Error('test: does not get device identifier');
   }
-
   return res.data;
 }
```

Comment:
```js
const sendMagicLink = async (email: string) => {
  const res = await authQuery.sendMagicLink(email);

  if (!res.data?.deviceIdentifier) {
    throw new Error('test: does not get device identifier');
  }
  
  setLocalConfig('deviceIdentifier', res.data.deviceIdentifier);
  return res.data;
}
```

Let's use an arrow function similar to others. Also, does the above refactor work?